submap_service: "/rex_gridmapping/get_submap"
map_frame_id: "/odom"
robot_frame_id: "/base_link"
min_update_rate: 2.0
map_center_x: 0.0
map_center_y: 0.0
map_length_x: 3.0
map_length_y: 3.0
footprint_yaw: 0.0
max_gap_width: 0.1
